keyboards/splitography/keymaps/default/keymap.c

```diff
@@ -0,0 +1,330 @@
+/* Copyright 2021 Alexis Jeandeau
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+// this is the style you want to emulate.
+//
+// To flash steno firmware
+// ▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔
+//   Reset keyboard or press hw reset button on base (hole)
+//
+//   cd qmk_firmware/keyboards/steno
+//   sudo make KEYMAP=sdothum dfu
+//
+//   sudo make clean          (good practice before flashing)
+//   sudo make KEYMAP=<...>   (to compile check)
+//
+// Package requirements (for arch linux)
+// ▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔
+//   avr-gcc-atmel
+//   avr-libc-atmel
+//   dfu-programmer
+//
+// Code
+// ▔▔▔▔
+//   This source is shamelessly based on the "default" steno layout
+
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "eeconfig.h"
+#include "keymap_steno.h"
+
+#include "config.h"
+#include "splitography.h"
+
+extern keymap_config_t keymap_config;
```

Comment:
only this include should be needed here.
```suggestion
#include "keymap_steno.h"
```